### DISCUSSION TITLE
New Issue template URL parameters for dropdowns

### DISCUSSION THREAD

**Discussion by publicarray** [community/community#9317] [Projects and Issues]
Hi,

Is there a way to fill out the dropdown menu (Device Architecture) from a URL parameter? 

e.g.: https://github.com/publicarray/spksrc/issues/new?assignees=&labels=bug&template=bug_report.yml&title=Example%20bug%20&package=Jellyfin&version=1.0&model=DS%20123&arch=x86_64&firmware=DSM%206.0

**Comment by MathiasReker** [comment]
No this is not possible
